### ISSUE TITLE
Temperature is ignored for ollama provider

### ISSUE THREAD

**Issue by chhu** [continuedev/continue#3203]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Win11
- Continue version: v0.8.60
- IDE version: 1.95.3
- Model: See below
- config.json:
  
{
  "models": [
    {
      "title": "Qwen2.5 72B",
      "model": "qwen2.5:72b-instruct-q4_1",
      "completionOptions": {
        "maxTokens": 4096,
        "keepAlive": -1
      },
      "apiBase": "http://localhost:11434",
      "provider": "ollama",
      "contextLength": 10000,
      "maxTokens": 10000,
      "keepAlive": -1,
      "temperature": "0",
    },
  ],
```

### Description

No impact of temperature. Answer to same question is always different. Increasing also does not change the "gibberishness" so I suspect it is ignored...
contextLength and keepAlive works as expected. 

### To reproduce

Setup an ollama model, set temperature to 0, repeat same question.

**Comment by HD787** [comment]
I was curious about this as well.

it seems you can get completion options to work if your format your config.json like this:

```
"models": [
    {
      "model": "AUTODETECT",
      "title": "Autodetect",
      "provider": "ollama"
    }
  ],
  "completionOptions": {
    "temperature": 0,
    // "seed": 422, (this won't work unless you add it as a completion option to the source code, but it makes it deterministic)
    "maxTokens": 600
  },
```
  
It's worth noting that I was unable to get Ollama to produce gibberish, the temperature didn't have a super identifiable effect on the output. It does definitely work as changing the temperature will produce different outputs even if the model is made to be deterministic.

if you want to check for yourself I would say the easiest way is to do it like [this](https://github.com/ollama/ollama/blob/main/docs/api.md#request-8)

**Comment by Patrick-Erichsen** [comment]
@HD787 this is correct, to configure temperature you need to set it via the `completionOptions` config: https://docs.continue.dev/reference#completionoptions
